quixstreams/app.py

```diff
@@ -604,10 +655,11 @@ def _quix_runtime_init(self):
         """
         Do a runtime setup only applicable to an Application.Quix instance
         - Ensure that "State management" flag is enabled for deployment if the app
-          is stateful and is running on Quix platform
+          is stateful and is running in Quix Cloud
         """
         # Ensure that state management is enabled if application is stateful
-        # and is running on Quix platform
+        # and is running in Quix Cloud
+        logger.info("APPLICATION IS CONNECTED TO THE QUIX PLATFORM")
```

Comment:
Do we need this logging code here? 
Looks like a leftover

Reply:
I had added this one so when the app actually runs (vs just init'ing it) it lets you know it's connecting to Quix. Might be overkill though?

Reply:
Oh, I see. 
I think it is overkill because it already logs the broker address here - https://github.com/quixio/quix-streams/blob/45f3e15f1515730e72ffc348bcbad86cda445c78/quixstreams/app.py#L759